### DISCUSSION TITLE
EdgeTX splash screen timeout

### DISCUSSION THREAD

**Discussion by kt4rj** [EdgeTX/edgetx#1068] [Q&A] [answered]
In OpenTX I was able to set how long the splash screen showed when powering on. I can't seem to find that option in EdgeTX. Is this possible or will it be possible in the future? Thank you for your time...

**Comment by pfeerick** [comment, marked as answer]
For which radio? Some radios, i.e. TX16S have never had that option - neither power on or splash screen delay - even on OpenTX. Only the power off delay. 

i.e. OTX 2.3.14 on TX16S
![image](https://user-images.githubusercontent.com/5500713/142107120-8a1dd5ad-c62a-4b1f-bf47-84f118dcc624.png)

vs OTX 2.3.14 on X9D+2019
![image](https://user-images.githubusercontent.com/5500713/142109785-9537a9fe-06e7-4753-9350-fa0f55d448b7.png)

**Reply by kt4rj** [reply]
I just recently switched to the TX16s from a QZ7. I didn't realize this would be a radio specific setting. Thank you for your time...
